datafusion/common/src/error.rs

```diff
@@ -510,26 +581,3 @@ mod test {
         assert_eq!(std::mem::discriminant(e), std::mem::discriminant(&exp),)
     }
 }
-
-#[macro_export]
-macro_rules! internal_err {
-    ($($arg:tt)*) => {
-        Err(DataFusionError::Internal(format!($($arg)*)))
-    };
-}
```

Comment:
This is removed?

Reply:
Its not used by anyone, instead new macro will be create for internal errors